symfony/ux-meta/1.0/config/packages/webpack_encore.yaml

```diff
@@ -0,0 +1,49 @@
+webpack_encore:
```

Comment:
This file should not be in the ux-meta recipe. It should stay managed fully by the recipe of the WebpackEncoreBundle IMO.

Reply:
You’re right! Only files that need to be different for UX should be here.